contrib/IECoreMantra/src/IECoreMantra/procedural/Procedural.cpp

```diff
@@ -225,7 +199,7 @@ VRAY_ieProcedural::render()
 	ScopedGILLock giLock;
 	try
 	{
-		object ieCore = g_mainModuleNamespace["IECore"];
```

Comment:
If it returns though, we won't be able to actually load the procedural, and nothing will render at all. But this happens every time, so the procedural would never load.